clowdapp.yaml

```diff
@@ -0,0 +1,123 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ros
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ros
+  spec:
+    envName: ${ENV_NAME}
+    dependencies:
+    - rbac
+    deployments:
+    - name: backend
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        minReplicas: 1
+        initContainers:
+          - command:
+            - sh
+            - -c
+              python -m manage db upgrade && python -m ros.api.main
```

Comment:
I think you need an extra `-` here.

Reply:
But, isn't it something like this - https://github.com/RedHatInsights/vulnerability-engine/blob/master/deploy/clowdapp.yaml#L230-L235

Reply:
Right. You can do it that way as well, but I don't think the current version works without that `- >-` line before the python code.